### ISSUE TITLE
add "next use" facilities to DES and to CostGraph

### ISSUE THREAD

**Issue by rjhanes** [NREL/celavi#3]
Ported from https://github.nrel.gov/aeberle/celavi/issues/195

This issue is being addressed in branch `issue195`

Previous commit on this issue is 8b70d49

**Comment by rjhanes** [comment]
Running commit bfd3e33 on `tiny-data` with artificially low recycling costs:

```
costgraph_parameters:
    sc_begin: manufacturing
    sc_end: 
    - landfilling
    - cement co-processing
    - next use
    cg_verbose: 1
    save_cg_csv: True
    finegrind_cumul_initial: 1.0
    finegrind_initial_cost: 10.0
    finegrind_revenue: 50.0
    finegrind_learnrate: -0.05
    finegrind_material_loss: 0.3
    coarsegrind_cumul_initial: 1.0
    coarsegrind_initial_cost: 10.0
    coarsegrind_learnrate: -0.05
    cg_update_timesteps: 12
```

produced the following blade counts:

![blade_count_facets](https://user-images.githubusercontent.com/18059389/133520661-6ae80a72-c0e2-4240-bd15-6d9c5ed978d8.png)

which looks promising.

Still need to test on `medium-data` and/or a filtered version of `develop` before merging.

**Comment by rjhanes** [comment]
Closed with PR #50
